src/test/java/com/gluonhq/richtext/undo/CmdManagerTests.java

```diff
@@ -12,11 +14,16 @@ public void executionRules() {
 
        StringBuilder text = new StringBuilder("Text");
        CommandManager<StringBuilder> commander = new CommandManager<>(text);
-       commander.execute( new TestCommand());
+       Assertions.assertTrue(commander.isUndoStackEmpty());
+       Assertions.assertTrue(commander.isRedoStackEmpty());
+
+       commander.execute(new TestCommand());
 
-       Assertions.assertEquals( 1,  commander.undoStack.size());
-       Assertions.assertEquals( 0,  commander.redoStack.size());
-       Assertions.assertEquals( "Text-redo",  commander.context.toString());
+       Assertions.assertEquals(1, commander.undoStack.size());
+       Assertions.assertEquals(0, commander.redoStack.size());
+       Assertions.assertFalse(commander.isUndoStackEmpty());
+       Assertions.assertTrue(commander.isRedoStackEmpty());
+       Assertions.assertEquals("Text-redo", commander.context.toString());
```

Comment:
I would avoid updating the current tests. Any new functionality should add its own unit test cases.